### PR TITLE
Experimental validation

### PR DESCRIPTION
**Pull request by arroyostack**
Marshmallow validation for User, Vet, Groomer and Walker.

### CODE REVIEW

**Review by hectorpd — DISMISSED** [review]
validation approve

**Review by hectorpd — APPROVED** [review]

File: src/app.py [hunk 1]
```diff
@@ -14,10 +14,6 @@
 from flask_jwt_extended import JWTManager
 
 
-
-
-
-
 #from models import Person
```
Comment (hectorpd) [inline src/app.py:18]: add

File: src/api/models.py [hunk 2]
```diff
@@ -1,85 +1,58 @@
...
 
+ma = Marshmallow()
+
 
 # USER MODEL
 class User(db.Model):
     __tablename__ = "user"
-    id = db.Column(db.BigInteger, default=uuid.uuid4().int >>
-                   (128 - 32), primary_key=True)
+    id = db.Column(db.BigInteger, primary_key=True)
```
Comment (hectorpd) [inline src/api/models.py:14]: add

File: src/api/models.py [hunk 3]
```diff
@@ -1,85 +1,58 @@
...
     email = db.Column(db.String(200), unique=True, nullable=False)
     name = db.Column(db.String(100), unique=False, nullable=False)
     last_name = db.Column(db.String(100), unique=False, nullable=True)
     phone_number = db.Column(db.String(150), unique=True)
     avatar = db.Column(db.String(2000), unique=False)
-    password = db.Column(db.String(100), unique=False, nullable=False)
     is_admin = db.Column(db.Boolean, default=False)
```
Comment (hectorpd) [inline src/api/models.py:20]: add

File: src/api/models.py [hunk 4]
```diff
@@ -1,85 +1,58 @@
...
     is_active = db.Column(db.Boolean(), default=False)
     created_at = db.Column(db.DateTime, default=datetime.utcnow)
-
+    password = db.Column(db.String(1000), unique=False, nullable=False)
```
Comment (hectorpd) [inline src/api/models.py:23]: add

File: src/api/models.py [hunk 5]
```diff
@@ -1,85 +1,58 @@
...
-            "last_name": self.last_name,
-            "phone_number": self.phone_number,
-            "avatar": self.avatar,
-            "is_active": self.is_active,
-            "is_admin": self.is_admin,
-            "created_at": self.created_at     
-        }
-      
-    # VET MODEL
+
```
Comment (hectorpd) [inline src/api/models.py:29]: add

File: src/api/models.py [hunk 6]
```diff
@@ -1,85 +1,58 @@
...
 
 class VetModel(db.Model):
     __tablename__ = "vet"
-    id = db.Column(db.BigInteger, default=lambda: uuid.uuid4().int >> (
-        128 - 32), primary_key=True)
+    id = db.Column(db.BigInteger, primary_key=True)
```
Comment (hectorpd) [inline src/api/models.py:33]: add

File: src/api/models.py [hunk 7]
```diff
@@ -1,85 +1,58 @@
...
     email = db.Column(db.String(150), unique=True, nullable=False)
-    password = db.Column(db.String(200), unique=False, nullable=False)
```
Comment (hectorpd) [inline src/api/models.py:48]: add

File: src/api/models.py [hunk 8]
```diff
@@ -1,85 +1,58 @@
...
     name = db.Column(db.String(100), unique=False, nullable=False)
     last_name = db.Column(db.String(100), unique=False, nullable=False)
     phone_number = db.Column(db.String(150), unique=True, nullable=False)
     company_name = db.Column(db.String(100), unique=False, nullable=True)
+    # address = db.Column(db.String(120), nullable=False)
```
Comment (hectorpd) [inline src/api/models.py:39]: add

File: src/api/models.py [hunk 9]
```diff
@@ -1,85 +1,58 @@
...
     description = db.Column(db.String(2000), unique=False, nullable=False)
     avatar = db.Column(db.String(2000), unique=True, nullable=False)
     services = db.Column(db.String(2000), unique=False, nullable=True)
     price_low = db.Column(db.Integer, unique=False, nullable=False)
     price_high = db.Column(db.Integer, unique=False, nullable=True, default=None)
     created_at = db.Column(db.DateTime, default=datetime.utcnow)
+    password = db.Column(db.String(1000), unique=False, nullable=False)
```
Comment (hectorpd) [inline src/api/models.py:46]: add

**Review by hectorpd — APPROVED** [review]

**Review by Romu9319 — APPROVED** [review]

**Pull request closed without merging**